src/Core/Services/MetadataProviders/SqlMetadataProvider.cs

```diff
@@ -1366,7 +1369,7 @@ private async Task ValidateDatabaseConnection()
                 = $"SELECT * FROM {queryPrefix}{SqlQueryBuilder.QuoteIdentifier(tableName)}";
             adapterForTable.SelectCommand = selectCommand;
 
-            DataTable[] dataTable = adapterForTable.FillSchema(EntitiesDataSet, SchemaType.Source, tableName);
+            DataTable[] dataTable = adapterForTable.FillSchema(EntitiesDataSet, SchemaType.Source, schemaAndTableName);
```

Comment:
if we set `schemaAndTableName = {queryPrefix}{SqlQueryBuilder.QuoteIdentifier(tableName)}`, we need not add a new argument to this function.